### ISSUE TITLE
Algo schedulers

### ISSUE THREAD

**Issue by theOGognf** [theOGognf/finagg#10]
Need to implement algo schedulers for scheduling entropy coefficient and learning rates. They need to be implemented slightly differently since the LR scheduler will need to interface with PyTorch optimizers while the entropy scheduler just needs to update an entropy coefficient.

- [x] Entropy scheduler
- [x] LR scheduler
